PlaidLink/Classes/PLDLinkBankMFALoginViewController.m

```diff
@@ -54,9 +55,11 @@ - (void)viewDidAppear:(BOOL)animated {
 - (void)didTapSubmit {
   [_view.submitButton showLoadingState];
   [self.view endEditing:YES];
-  NSDictionary *options = @{
-      @"list" : @(YES)
-  };
+  
+    
+  NSMutableDictionary* options = [_options mutableCopy];
```

Comment:
options may be nil - recommend either initializing or checking here. also * is on the wrong side, should be NSMutableDictionary *options